src/js/modules/articles/selectors.js

```diff
@@ -63,10 +74,10 @@ export const getArticleFromRequestedSlug = createSelector(
 );
 
 /**
- * The selector returns a filtered articles object that contains all articles
+ * The selector returns an object that contains all articles
  *   written by a contributor.
  */
-export const getArticlesByContributor = createSelector(
+export const getContributorArticles = createSelector(
```

Comment:
`getRequestedContributorArticles`?  `getContributorArticles` is a little confusing

Reply:
????

Reply:
W/e you can leave it. But I'd still like a better, clearer explanation